src/RTTI/RTTI_EventMessages.hpp

```diff
@@ -0,0 +1,157 @@
+#include <AI/EventMessage.hpp>
+#include <RTTI/RTTIUtil.hpp>
+
+namespace REGoth
+{
+  namespace AI
+  {
+    using UINT32 = bs::UINT32;
```

Comment:
Didn't think of this idea to circumvent `using namespace bs` :D We should definitely open a PR to the bs::f repo in the future to really fix the issue.